src/hooks/core/chat/useFetchChannelHook.ts

```diff
@@ -107,7 +88,7 @@ const useFetchChannelHook = () => {
         (channel?.messages || []).map(async (message) => {
           if (message?.type === 'deleted') return;
           if (message?.type === 'system') {
-            message = helperFollowingSystemMessage(message);
+            message = getFirstMessage([message]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The replacement of the old helper function with the new `getFirstMessage` function from the `useSystemMessage` hook is a good change. It makes the code cleaner and more maintainable.
```diff
+ message = getFirstMessage([message]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->